src/wrap.coffee

```diff
@@ -29,15 +30,32 @@ class Wrap
 			headers: @_client.getAuthHeader()
 			body
 		}
-		return http.post("#{@_client.baseUrl}/wraps/#{@id}/personalize", options)
+		return http.post("#{@_wrapUrl}/personalize", options)
 
-	personalize: (schemaMap) ->
+	listPersonalized: (search) ->
+		options = {
+			headers: @_client.getAuthHeader()
+			search
+		}
+		return http.get("#{@_wrapUrl}/personalize", options)
```

Comment:
I'd argue these might be easier to understand with `options` just inlined here. The function then becomes a single statement one which is as basic as you can get. In addition, I'd say the name `options` doesn't add any meaningful information. Same for other similar methods.